src/python/deepgnn/pytorch/training/factory.py

```diff
@@ -117,7 +118,7 @@ def run_dist(
 
     num_workers = (
         0
-        if issubclass(dataset.sampler_class, (GENodeSampler, GEEdgeSampler))
+        if issubclass(dataset.sampler_class, (GENodeSampler, GEEdgeSampler)) or platform.system() == "Windows"
```

Comment:
Do we need to run examples in CI on windows to trigger this behavior?

Reply:
Yes the original bug has been here for all deepgnn package versions. Should we add powershell run files to all the examples or install something so we can use bash in windows?